### DISCUSSION TITLE
PS2 Keyboard

### DISCUSSION THREAD

**Discussion by NateHubbard** [slu4coder/Minimal-64x4-Home-Computer#32] [Q&A] [answered]
When the keyboard is plugged in I just get a continuous stream of "v" over and over and the keyboard does nothing else.
The keyboard is a Perixx Periboard-409 that claims to be 5V. In the schematic it mentions both the HC132 and the HCT132 (which is what I have). Is the HCT132 correct for a 5V keyboard? Because it sounds like maybe I have the wrong chip, or maybe just a bad one?
Pulling the HCT132 completely and plugging in the keyboard stops the "vvvvvvv" issue, and the keyboard lights up with power like it's at least on. Obviously, you can't type.
Otherwise I feel like I was so close to having a working machine.

**Comment by nateo87** [comment, marked as answer]
I had the same issue originally - this is because more recent PS/2 keyboards attempt to do a "handshake" with whatever device they're plugged into. Older (think early 2000's) keyboards don't behave this way. Check out this closed issue from a few months ago:
https://github.com/slu4coder/Minimal-64x4-Home-Computer/issues/15

**Reply by NateHubbard** [reply]
Well crap. You know, I think I remember seeing something about well before I actually soldered anything, but didn't remember the details.
Thanks though, this makes it sound like an easy fix is to just grab an old keyboard from somewhere.

**Reply by nateo87** [reply]
I just went out to a local thrift store and was able to find an old Dell keyboard for around $5. Worked a treat.

**Comment by NateHubbard** [comment]
I just tried an early 90s Tandy, 101 key PS2 keyboard (the same model I used in high school three decades ago) and it's just spewing 'vvvvvvv', the same as a newer keyboard. 
Age does not seem to be the issue here. I'm beginning to suspect that price (at the time) might have been the actual issue and maybe cheaper keyboards would work.
Any recommendations, including maybe at exact model number?

**Comment by NateHubbard** [comment]
Ok, I got a couple of cheap old keyboards from my brother and both of those work, thankfully. An NEC and something called a KB915B.
